src/app/view/components/listing-data-table/listing-data-table.component.scss

```diff
@@ -26,7 +29,8 @@
             }
         }
 
-        .ui-datatable-data {
+        .ui-datatable-data,
```

Comment:
I think we can get rid of `.ui-datatable-data` since `listing-data-table.component` will not use `p-table` component.